spark-publish/index.js

```diff
@@ -66,13 +65,11 @@ export const publish = async ({
   const ieAddMeasurementsDuration = new Date() - start
   logger.log('Measurements added to round', roundIndex.toString())
 
-  // Mark measurements as shared
+  // Delete published measurements
```

Comment:
I had to include the commit from #186 in this PR to make the tests pass. I'll rebase this branch after #186 is landed to remove that commit.

The changes are best to be reviewed in this view:
https://github.com/filecoin-station/spark-api/pull/187/commits/ce3815b9e80057984bbc4e5f5163c2bfb0fbe3c6